eskip/eskip.go

```diff
@@ -20,6 +20,20 @@ var (
 	duplicateMethodPredicateError   = errors.New("duplicate method predicate")
 )
 
+type DefaultFilters struct {
+	Prepend []*Filter
+	Append  []*Filter
+}
+
+func (df *DefaultFilters) Do(r []*Route) []*Route {
```

Comment:
it would be nice to have a test that the filters are prepended and appended

Reply:
done